shared/style/contacts/contacts_buttons.css

```diff
@@ -16,7 +16,7 @@ li button.send-sms {
 }
 li button.icon-email:before {
   background-image: url("/shared/style/contacts/images/mail.png") !important;
-} 
+}
 
```

Comment:
most likely it was lint error, usually trailing spaces are treated as errors/warnings